src/modules/timeAgo/index.ts

```diff
@@ -1,6 +1,102 @@
-import TimeAgo from "./timeAgo";
+import { digitsFaToEn } from "../digits";
 
-export default (datetime: string): string => {
-	const timeago = new TimeAgo(datetime);
-	return timeago.getTimeAgo();
-};
+export function convertToTimeStamp(datetime: string): number {
+	const patternDateTime = /(\d+)-(\d+)-(\d+) (\d+):(\d+):(\d+)/;
+	const dateTime = datetime.match(patternDateTime);
+	if (dateTime) {
+		const date: Date = new Date(
+			Number(dateTime[1]),
+			parseInt(dateTime[2], 10) - 1,
+			Number(dateTime[3]),
+			Number(dateTime[4]),
+			Number(dateTime[5]),
+			Number(dateTime[6]),
+		);
+		return date.getTime(); // 1623036870000
+	} else throw new TypeError("PersianTools: timeAgo - The input format must be yyyy-mm-dd hh:mm:ss");
+}
+
+/**
+ * Get current timestamp of Now datetime
+ */
+export function getTimeNow(): number {
+	const date: string[] = new Date(Date.now())
+		.toLocaleDateString("fa-IR", {
+			year: "numeric",
+			month: "2-digit",
+			day: "2-digit",
+		})
+		.split("/");
+
+	const time: string[] = new Date(Date.now())
+		.toLocaleTimeString("fa-IR", {
+			hour: "2-digit",
+			minute: "2-digit",
+			second: "2-digit",
+		})
+		.split(":");
+
+	return convertToTimeStamp(
+		digitsFaToEn(date[0] + "-" + date[1] + "-" + date[2] + " " + time[0] + ":" + time[1] + ":" + time[2]),
```

Comment:
it does not make sense that what is `date[x]` and `time[x]`. use the Array destructuring feature and use good names.

Reply:
I rebuilt this function and remove `date[x]` and `time[x]`.